ironfish/src/migrations/data/023-wallet-optional-spending-key.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
review migration

Reply:
It will be a race between this and #3515 to see which gets Migration023!

Reply:
Looks good. We should test run it.,